docs/index.md

```diff
@@ -60,13 +65,22 @@ provider "onepassword" {
 }
 ```
 
+Each of the configuration options can be set using env variables.
+That's more secure way to do that as prevents from accidentally pushing sensitive values to the project repo.
+
+- `OP_CONNECT_HOST` - The URL where your 1Password Connect API can be found.
```

Comment:
In the Connect documentation, we typically refer to this environment variable as the Connect server's URL/hostname/IP (not the Connect API)... I know it's because we're accessing the API, but I think we should keep it consistent 🙂